### PR TITLE
Move CI from Travis to GitHub Actions

### PR DESCRIPTION
**Pull request by sunny**
As suggested in #27, this uses GHA instead of Travis, following previous work on https://github.com/dblock/iex-ruby-client/pull/117

### CODE REVIEW

**Comment by sunny** [comment]
@dblock GHA needs to be enabled to be able to run here 🏃🏻

File: README.md [hunk 1]
```diff
@@ -2,7 +2,6 @@ OpenWeather Ruby Client
 =======================
 
 [![Gem Version](https://badge.fury.io/rb/open-weather-ruby-client.svg)](https://badge.fury.io/rb/open-weather-ruby-client)
-[![Build Status](https://travis-ci.org/dblock/open-weather-ruby-client.svg?branch=master)](https://travis-ci.org/dblock/open-weather-ruby-client)
```
Comment (dblock) [inline README.md:5]: Can we please get a badge in the next PR?

**Comment by dblock** [comment]
Any interest in co-maintaining this library @sunny? If yes drop me a note to dblock at dblock dot org with your rubygems access, you could make the next release after #27.

**Comment by sunny** [comment]
Thanks for the suggestion, however I won’t be a good fit for maintaining this as I am not using the gem in my current projects.

**Comment by dblock** [comment]
Thanks for your help @sunny !

**Pull request merged**